charts/nauth/templates/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "nauth.name" . }}
+  name: {{ include "nauth.fullname" . }}
```

Comment:
I would also suggest using `nauth.fullname` for all other resources or their prefixes.